arbeitszeit_flask/templates/macros/plan_details.html

```diff
@@ -149,6 +149,14 @@ <h1 class="title">
                         </p>
                     </div>
                 </div>
+                {% if view_model.details.type_of_plan[1] == 'Public' %}
```

Comment:
done. Furthermore, I think when always displaying both information it would be helpful to elaborate a little bit on the context, i.e. why the info is there and/or what it means (especially for newbie users like me). This applies to other fields as well. Maybe this could also be part of or another follow-up PR and could be done using these (i) icons that you can click or hover?

Reply:
done here: https://github.com/arbeitszeit/arbeitszeitapp/issues/954